cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,41 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until database is available."""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        attempts = 0
+        max_attempts = 30
+        wait_time = 1
+
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                db_conn.cursor()
+            except OperationalError:
+                attempts += 1
+                if attempts >= max_attempts:
```

Comment:
This line checks if the maximum number of attempts to connect to the database has been reached. If it has, an error message will be printed and an `OperationalError` will be raised. This is a good way to prevent the application from getting stuck in an infinite loop if the database is not available.